### PR TITLE
change new register redirect

### PR DESCRIPTION
**Pull request by toprasit2**

### CODE REVIEW

File: live-teaching-web/src/app/register/register.component.ts [hunk 1]
```diff
@@ -29,14 +29,13 @@ export class RegisterComponent implements OnInit {
...
-    private messageService: MessageService
   ) {}
 
   ngOnInit() {
     this.afAuth.user.subscribe(user => {
       if (user) {
-        this.userAuth = user.providerData[0];
-        let id = this.userAuth.email.split("@psu.ac.th")[0];
+        this.userAuth = user;
+        let id = this.userAuth.providerData[0].email.split("@psu.ac.th")[0];
```
Comment (new-bmk) [inline live-teaching-web/src/app/register/register.component.ts]: If providerData is `null`. bug ?

**Review by new-bmk — CHANGES_REQUESTED** [review]
Please, check comment.

File: live-teaching-web/src/app/register/register.component.ts (hunk 1 above)
Comment (toprasit2) [reply live-teaching-web/src/app/register/register.component.ts]: already check before route to this page but not use now

File: live-teaching-web/src/app/register/register.component.ts [hunk 2]
```diff
@@ -28,27 +28,27 @@ export class RegisterComponent implements OnInit {
...
     private afAuth: AngularFireAuth,
-    private authService: AuthService,
+    private authService: AuthService
   ) {}
 
   ngOnInit() {
     this.afAuth.user.subscribe(user => {
       if (user) {
         this.userAuth = user;
-        let id = this.userAuth.providerData[0].email.split("@psu.ac.th")[0];
```
Comment (new-bmk) [inline live-teaching-web/src/app/register/register.component.ts]: Are you sure?. that `providerData` always exists.
but I will approve before.

**Review by new-bmk — APPROVED** [review]

**Pull request merged**